MSAL/test/app/mac/MSALAcquireTokenViewController.m

```diff
@@ -467,4 +524,54 @@ - (MSALAccount *)selectedAccount
     return self.accounts[self.userPopup.indexOfSelectedItem-1];
 }
 
+- (MSALInteractiveTokenParameters *)tokenParams:(BOOL)isSSOSeedingCall
+{
+    MSALTestAppSettings *settings = [MSALTestAppSettings settings];
+    MSALInteractiveTokenParameters *parameters = [[MSALInteractiveTokenParameters alloc] initWithScopes:isSSOSeedingCall ? [self getSSOSeedingScope] : [settings.scopes allObjects]
+                                                                                      webviewParameters:[self msalTestWebViewParameters]];
+    
+    if (isSSOSeedingCall)
+    {
+        parameters.authenticationScheme = [MSALAuthenticationSchemeBearer new];
+    }
+    else
+    {
+        parameters.authenticationScheme = [self authScheme];
+    }
+    
+    parameters.loginHint = [self.loginHintTextField stringValue].length ? [self.loginHintTextField stringValue] : nil;
+    parameters.account = settings.currentAccount;
+    parameters.promptType = isSSOSeedingCall ? MSALPromptTypeDefault : [self promptType];
+    
+    parameters.extraQueryParameters = isSSOSeedingCall ? [NSDictionary msidDictionaryFromWWWFormURLEncodedString:@"prompt=none"] : [NSDictionary msidDictionaryFromWWWFormURLEncodedString:[self.extraQueryParamsTextField stringValue]];
+//    parameters.extraQueryParameters = isSSOSeedingCall ? [NSDictionary new] : [NSDictionary msidDictionaryFromWWWFormURLEncodedString:[self.extraQueryParamsTextField stringValue]];
+
+    return parameters;
+}
+
+- (NSArray<NSString *> *)getSSOSeedingScope
```

Comment:
I believe this is used across iOS and macOS. Can we put it to settings file to avoid duplicate code perhaps?

Reply:
Addressed.